CONTRIBUTING.md

```diff
@@ -33,31 +33,31 @@ When contributing to this repository, please first discuss the change you wish t
 
 - Push your changes:  ```git push```
 
-- Send a PR to ods-ci using GitHub's web interface
+- Send a PR to rhoda-ci using GitHub's web interface
 
 - If the PR can't be merged, rebase your branch to master:
   ```
-  git remote add upstream https://github.com/red-hat-data-services/ods-ci
+  git remote add upstream https://github.com/RHODA-lab/rhoda-ci.git
   git fetch upstream
   git checkout add-test-ods-542-alerts
   git rebase upstream/master
   git push -f
   ```
 
-- Test your PR in Jenkins using the rhods-ci-pr-test pipeline
-   - https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test
-   - Log in if required
-   - Build with Parameters (if you don't see this option contact the QE team)
-     - Set the PR id (e.g. 42) in ODS_GIT_REPO_PULL_REQUEST_ID
-     - Select TEST_CLUSTER
-     - Build
+#- Test your PR in Jenkins using the rhoda-ci-pr-test pipeline
```

Comment:
commented this block for now, which will be included when we defined our pipeline